scripts/tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+echo "mode: count" > /tmp/coverage.out
+
+for d in $(go list ./... | grep -v vendor); do
```

Comment:
You can use [glide nv](https://github.com/Masterminds/glide#glide-novendor-aliased-to-nv) `for d in $(glide nv)`